RELEASES.md

```diff
@@ -57,6 +58,112 @@ Get acquainted with the latest EDP releases.
+
+A new release brings a new look. Check out our new [documentation site](https://docs.kuberocketci.io/), which features improved navigation and documentation versioning support.
+
+We have updated terminology within the platform. Environments are now referred to as Deployment Flows, and Stages are now Environments. These changes align our terminology with globally recognized terms, reducing user confusion. For more details, please refer to our updated [Glossary](https://docs.kuberocketci.io/docs/next/glossary).
+
+We have added the ability to specify custom build and review pipelines with just one click when editing or creating new codebase branches. This enhancement increases flexibility when working with codebases, allowing for easier modifications to pipeline logic based on specific requirements.
+
+Another significant enhancement is the introduction of a new section called Pipelines. This section centralizes pipeline observability by consolidating all pipelines in one place. We are continuing to migrate Tekton functionality into our platform. Consequently, we have consolidated our pipelines into a single location. The new Pipelines section displays the entire Tekton Pipeline Run history, simplifying pipeline management and observability.
+
+We are expanding the options for managing applications. Users are now able to automate the deployment of their applications packaged as RPMs, utilizing Ansible for this process.
+
+As part of our expanding language support, we’ve added C and C++ to our CI/CD platform. This includes integration with the make and CMake build tools, enhancing the platform's capabilities for building and managing C/C++ projects.
+
+We have also introduced the ability to [customize Environment deletion](https://docs.kuberocketci.io/docs/next/operator-guide/cd/customize-environment-deletion). This feature is particularly useful for users with applications that have specific configurations or dependencies, e.g. database cleanup, infrastructure tearing down, etc. Customized clean flows make it easier to maintain components throughout their lifecycle.
+
+Additionally, we have added new functionality to our deployment pipelines. A new tab called Variables has been added to the Environment (formerly Stage) details page. In this tab, users can specify key-value parameters, which will be transmitted within the application as environment variables. This improvement allows users to configure application deployment settings without leaving the KubeRocketCI portal.
+
+Deployment pipelines have been enhanced with a manual approval feature. This allows developers to integrate a checkpoint within their custom deployment pipelines, requiring a manual review and approval process before the pipeline can advance to the next stage. This ensures greater control and oversight over the deployment process.
```

Comment:
We are pleased to introduce a new Manual Approval feature on the platform. This feature allows developers to incorporate an approval task within their custom deployment pipelines, requiring a manual review and approval before advancing to the next stage. While deployment pipelines are a primary use case for this capability, the approval task can also be leveraged in other workflows. This enhancement provides greater control and oversight, ensuring all processes meet required standards and criteria before progressing.